src/domain/event/queryUtils.ts

```diff
@@ -72,46 +72,63 @@ export const useSimilarEventsQuery = (
   return { data, loading };
 };
 
-const useOtherEventTimesVariables = (
-  event: EventFields,
-  eventType: EventTypeId = EventTypeId.General
-) => {
+const useOtherEventTimesVariables = (event: EventFields) => {
   const superEventId = React.useMemo(
     () => getEventIdFromUrl(event.superEvent?.internalId || '', 'event'),
     [event.superEvent]
   );
 
   const variables = React.useMemo(
     (): EventListQueryVariables => ({
-      include: ['keywords', 'location'],
+      // include: ['keywords', 'location'],
       sort: EVENT_SORT_OPTIONS.START_TIME,
       start: 'now',
       superEvent: superEventId,
-      eventType,
+      // TODO: Since LE v2 is listing general type of events 
+      // when no event type is given as a parameter,
+      // this needs a list of eventTypes, 
+      // or otherwise some events or courses are always excluded.
+      // eventType: EventTypeId.Course //event.typeId,
     }),
-    [eventType, superEventId]
+    [superEventId]
   );
 
   return { superEventId, variables };
 };
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const useOtherEventTimes = (
+export const useSubEventsQueryVariables = (
+  event: EventFields
+): { superEventId: string | undefined; variables: EventListQueryVariables } => {
+  const variables = React.useMemo(
+    (): EventListQueryVariables => ({
+      // include: ['keywords', 'location'],
+      sort: EVENT_SORT_OPTIONS.START_TIME,
+      start: 'now',
+      superEvent: event.id,
+      // TODO: Since LE v2 is listing general type of events 
+      // when no event type is given as a parameter,
+      // this needs a list of eventTypes
+      // or otherwise some events or courses are always excluded.
+      // eventType: EventTypeId.Course //event.typeId
+    }),
+    [event.id]
+  );
+
+  return { superEventId: event.id, variables };
+};
```

Comment:
I'm not sure if the is need for this helper function

Reply:
No there is no need for it. I just "started from somewhere" and made it equal with the old implementation of "other times of the event". I'll remove those both. 👍

Reply:
Actually, after rethinking this, I think I keep them both there, because I really need `useOtherEventTimesVariables` and it has several dependencies. I would not like to import those dependencies in subEvents and like this its easy to see what is the difference between those 2. I can move them if you don't agree

Reply:
Thinking if React.useMemo is at all useful here. `useOtherEventTimes` could do without both useMemos and make code easier. I don't think there is any noticeable performance difference